unchanged:
--- /dev/null
+++ b/.octaverc
@@ -0,0 +1 @@
+pkg load image
unchanged:
--- a/SLIC.cpp
+++ b/SLIC.cpp
@@ -1,5 +1,5 @@
 #include "mex.h"
-#include "matrix.h"
+//#include "matrix.h"
 #include <cfloat>
 #include <cmath>
 #include <iostream>
@@ -43,7 +43,7 @@ public:
 	//======================================================================
 	//mean rgb in each superpixel
 	//=======================================================================
-	void SLIC::DoMeanSup( double *  &                    m_rr,
+	void DoMeanSup( double *  &                    m_rr,
 	                      double *  &                    m_gg,
                   	      double *  &                    m_bb,
 	                      int &                          numlabels,
@@ -1012,4 +1012,4 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
 	
 	slic.DoRGBtoLABConversionSup(meanSuprr, meanSupgg, meanSupbb, Supll, Supaa, Supbb, numlabels); 
 	
-}
\ No newline at end of file
+}
diff -u b/main.m b/main.m
--- b/main.m
+++ b/main.m
@@ -2,18 +2,20 @@
 close all
 clc
 %Define Ground Truth (GT) and Image (IM) Paths
-path_IM='D:\SALIENCY_BENCHMARKS_AND_COMPARISONS\SALIENCY_BENCHMARKS\DUTUMRON5168\Images\';
+path_IM='./images/';
 
 %Define Output Folder (OUT) Path 
-path_OUT='C:\Users\aytekin\Desktop\EQCUT_SHARE\TESTS\DUTOMRON\';
+path_OUT='./output/';
 
 %Change the .jpg extension accordingly
 contents=dir([path_IM '*.jpg']);
 
+fprintf('Number of images found: %d \n', length(contents));
 
 for i=1:length(contents)
     name=contents(i).name;
     name=name(1:end-4);
+    fprintf('Converting %s: ', name);
     imname=strcat(path_IM,name,'.jpg');
     image_now=(imread(imname));
     
@@ -47,4 +49,5 @@
     %Save the result
     imwrite(SalMapFinal,[path_OUT name '_EQCUT.png']);
+    fprintf('Done\n');
     
 end
